client/app/scripts/liveblog-edit/module.js

```diff
@@ -56,6 +56,7 @@ define([
 
         // define the $scope
         angular.extend($scope, {
+            publishDisabled: true,
```

Comment:
What is it for ? I don't see this variable used somewhere. Maybe did you forgot to use it in the view ?